src/test/kotlin/no/nav/syfo/mocks/NorgMock.kt

```diff
@@ -0,0 +1,60 @@
+package no.nav.syfo.mocks
+
+import com.auth0.jwt.JWT
+import io.ktor.client.engine.mock.*
+import io.ktor.client.request.*
+import io.ktor.http.*
+import no.nav.syfo.application.api.auth.JWT_CLAIM_NAVIDENT
+import no.nav.syfo.client.norg.domain.NorgEnhet
+import no.nav.syfo.testhelper.UserConstants
+
+private val norgEnhet = NorgEnhet(
+    enhetNr = UserConstants.ENHET_OVERORDNET,
+    navn = "enhet",
+    status = "aktiv",
+    aktiveringsdato = null,
+    antallRessurser = null,
+    kanalstrategi = null,
+    nedleggelsesdato = null,
+    oppgavebehandler = null,
+    orgNivaa = null,
+    orgNrTilKommunaltNavKontor = null,
+    organisasjonsnummer = null,
+    sosialeTjenester = null,
+    type = null,
+    underAvviklingDato = null,
+    underEtableringDato = null,
+    versjon = null,
+)
+
+private val norgResponse = listOf(norgEnhet)
+
+fun MockRequestHandleScope.getNorgResponse(request: HttpRequestData): HttpResponseData {
+    val token = request.headers[HttpHeaders.Authorization]?.removePrefix("Bearer ")
+    val veilederIdent = JWT.decode(token).claims[JWT_CLAIM_NAVIDENT]?.asString()
```

Comment:
Finnes kanskje en util for å hente veilederIdent fra token?

Reply:
Det ligger util-metoder for dette i PipelineUtil i mange av de andre app'ene.

Reply:
Jeg fjerner det, siden det ikke er i bruk 👍 
Vi har utils for dette her også, men de funker ikke i mocken fordi de er extensions på `ApplicationCall`, ikke `HttpRequestData` 🤷